nixos/modules/services/web-servers/gunicorn/instance-options.nix

```
@@ -0,0 +1,112 @@
+{
+  config,
+  lib,
+  ...
+}:
+
+{
+  options = {
+    app.package = lib.mkOption {
+      type = lib.types.package;
+      description = "Package of the WSGI Python application.";
+    };
+
+    app.python = lib.mkOption {
+      type = lib.types.package;
+      default = config.app.package.python;
+      defaultText = lib.literalExpression "services.gunicorn.instances.<name>.app.package.python";
```

See other comment about `passthru.wsgiModule`, goes for this one as well.

```suggestion
      defaultText = lib.literalExpression "config.services.gunicorn.instances.<name>.app.package.python";
```